Makefile

```diff
@@ -11,13 +11,23 @@ up:
 	@docker compose up -d
 
 log:
-	@docker compose logs -f --tail 10000 $(service)
+	@if [ -z "$(service)" ]; then \
+		docker compose logs -f --tail 10000; \
+	else \
+		docker compose logs -f --tail 10000 --no-log-prefix $(service); \
+	fi
 
 build:
 	@docker compose build
 
 shell:
-	@docker compose run --rm --no-deps $(service) sh
+	@docker compose exec $(service) sh || { \
+		echo "\033[38;5;214m[!] Fallback to 'docker compose run'\033[0m"; \
+		docker compose run --rm --no-deps $(service) sh; \
+	}
+
+status:
+	@docker compose ps -a --format "{{.Label \"com.docker.compose.service\"}}|{{.Status}}" $(service) | column -t -s "|"; \
```

Comment:
shows containers in the compose that are running, exited etc. only the name of the service and the status as more just makes it harder to read and doesn't add much value imo.